src/client/components/Appheader/AppHeader.component.js

```diff
@@ -0,0 +1,20 @@
+// dumb/statless img component with props
+
+import React from 'react';
+import './AppHeader.style.css';
+import PropTypes from 'prop-types';
+
+export const AppHeader = (props) => {
+  return (
+    <div className="AppHeader-div">
+      <img src={props.path} alt="AppHeader" />
```

Comment:
then you could also do this
```suggestion
      <img src={src} alt="AppHeader" />
```